packages/core/src/v3/utils/ioSerialization.ts

```
@@ -348,14 +352,18 @@ export async function createPacketAttributesAsJson(
 
   switch (dataType) {
     case "application/json":
-      return imposeAttributeLimits(flattenAttributes(data, undefined));
+      return imposeAttributeLimits(
+        flattenAttributes(data, undefined, OTEL_SPAN_ATTRIBUTE_COUNT_LIMIT)
+      );
     case "application/super+json":
       const { deserialize } = await loadSuperJSON();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)